src/migration.ts

```diff
@@ -6,7 +6,7 @@ import * as fs from 'fs/promises'
 import { exists } from './utils'
 
 interface LegacyState {
-    selectedManifest: { uri: vscode.Uri } | null
+    selectedManifest: { uri: { path: string } } | null
```

Comment:
Can't this be `selectedManifest?: {uri: {path: string }},` instead?

Reply:
It was using null before, so it may cause problems if it now maps to undefined